docs/user/unofficial-projects.rst

```diff
@@ -0,0 +1,134 @@
+Policy for Unofficial and Unmaintained Projects
+===============================================
+
+This policy describes a process where we take actions against unmaintained_ and unofficial_ forks of project documentation.
+
+
+Rationale
+---------
+
+Documentation projects may be kept online indefinitely, even though a newer version of the same project exists elsewhere.
+There are many reasons this can happen,
+including forks, old official docs that are unmaintained, and many other situations.
+
+The problem with old, outdated docs is that users will find them in search results,
+and get confused to the validity of them.
+Projects will then get support requests from people who are using an old and incorrect documentation version.
+
+We have this policy to allow project maintainers to request the *delisting* of forks that are old and outdated.
+
+
+Examples
+~~~~~~~~
+
+Common cases covered by this policy are:
+
+Outdated docs
+    Documentation that is not being actively maintained or updated. This means for instance that the source project has updates that are not reflected in the documentation or a newer official documentation exists elsewhere on the internet.
+
+Unofficial forks
+    Forks of a project documentation that are not linked to from the project in any way.
+
+
+Specification
+-------------
```

Comment:
```suggestion
-------------

High level overview
~~~~~~~~~~~~~~~~~

The process at a high level looks like:

* A reporter emails us about a project they think is outdated and unofficial
* An RTD team member evaluates it to make sure it's outdated and unofficial, according to this policy
* We delist this project from search results and send an email to owners of the RTD project
* If a project owner objects, we evaluate their evidence and make a final decision
```

I wonder if a tl;dr is useful?

Reply:
I think this looks useful, at the very least for the core team to grasp what is supposed to happen since the procedure might evolve to be a tad bit more confusing :+1:

Reply:
Yea, I also think users in general don't understand how the abandoned projects policy works. We probably need to rename that one in some fashion (eg. Policy for changing project slug?) since it's not obvious what the diff is between abandoned & unmaintained/unofficial.

Reply:
That's a good point. Over-all they fit into some kind of category of "what can I do if I see a documentation project on RTD that I think should be moved/removed/unlisted". But I guess the best fit will be a "Policy" sub-level in our future "Reference" category. We just need to label them well.

"what can I do if I see a documentation project on RTD that I think should be moved/removed/unlisted?" might fit better in FAQ, albeit I feel some kind of temptation to advocate against FAQs because Diátaxis should provide us with the structure to avoid them :)